lambdas/preview/index.py

```diff
@@ -141,35 +145,102 @@ def extract_vcf(file_):
     """
     Pull summary info from VCF: meta-information, header line, and data lines
     (in that order, up to MAX_LINES)
-    """
 
+    Args:
+        file_ - file-like
+    """
```

Comment:
Throw a returns string in here. Not usually a big deal, but with all the string vs bytes going on it helps.